package.json

```diff
@@ -30,6 +30,11 @@
     "canjs-plugin",
     "donejs"
   ],
+  "steal": {
+    "plugins": [
+      "steal-css"
```

Comment:
If I add `steal-qunit` as a plugin I get this error:
```
GET http://localhost/can-util/node_modules/steal-qunit/node_modules/steal-css/package.json 404 (Not Found)
```
Adding `steal-css` made it go away, so that's what I did. What's the correct solution?